save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/ScoreCard.kt

```diff
@@ -17,11 +17,11 @@ import kotlinx.js.jso
 /**
  * ProjectScoreCardProps component props
  */
-external interface ProjectScoreCardProps : Props {
+external interface ScoreCardProps : Props {
     /**
-     * Name of a current project
+     * Name of a current project or contest
```

Comment:
> project or contest

Kind of confusing

Reply:
The point is that this component is used either to display the results of a given project in some contest (in this way the name of a contest is shown) or the results in a given contest (in this way the name of a project is shown)

Reply:
Yes, no need to create a separate comnponent, but maybe at least explain a bit more:
```suggestion
     * Name of a current project or contest (acts as a card header)
```